.github/workflows/release.yml

```
@@ -0,0 +1,254 @@
+      # GitHub credentials in order to open a PR. However, the `brew tap` command
+      # must be run for Homebrew to see the dir as a tap.
+      # This is necessary to avoid the following error:
+      # Error: No available formula or cask with the name "mas-cli/tap/mas".
+      - name: 🚰 Checkout mas tap
+        run: |
+          rm -rf /opt/homebrew/Library/Taps
+          mkdir Taps
+          ln -s $(realpath Taps) /opt/homebrew/Library/Taps
+          ls -l /opt/homebrew/Library/Taps
+          brew tap mas-cli/tap
+          brew tap
+
+      # Checkout with a token for write access
+      - name: 🚰 Re-clone mas tap
+        uses: actions/checkout@v4
+        with:
+          repository: mas-cli/homebrew-tap
+          token: ${{ secrets.DEPLOY_TOKEN }}
+          # brew --repository mas-cli/tap
+          path: Taps/mas-cli/homebrew-tap
+
+      - name: 🔧 Configure Git Author
+        run: |
+          git config --global user.name 'masclibot'
+          git config --global user.email 'masclibot@users.noreply.github.com'
+
+      - name: ⤴️ Open mas tap PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
+          DRY_RUN: ${{ needs.start.outputs.dry_run }}
+          RELEASE_BRANCH: ${{ needs.start.outputs.release_branch }}
+          MAS_VERSION: ${{ needs.start.outputs.mas_version }}
+        working-directory: /opt/homebrew/Library/Taps/mas-cli/homebrew-tap
+        run: |
+          git branch "${RELEASE_BRANCH}"
+          git switch "${RELEASE_BRANCH}"
+
+          cp -v \
+            ${GITHUB_WORKSPACE}/Homebrew/mas-tap.rb \
+            Formula/mas.rb
+
+          git add Formula/mas.rb
+          git commit --message="🔖 Version ${MAS_VERSION}"
+
+          git push \
+            --set-upstream origin "${RELEASE_BRANCH}"
+
+          gh pr create \
+            --assignee phatblat \
+            --base main \
+            --head "${RELEASE_BRANCH}" \
+            --title "🔖 Version ${MAS_VERSION}" \
+            --body "This PR contains the changes from releasing version \
+              [${MAS_VERSION}](https://github.com/mas-cli/mas/releases/tag/${MAS_VERSION})."
+
+      - name: 🚰 Update mas tap formula
+        env:
+          GH_TOKEN: ${{ github.token }}
+          DRY_RUN: ${{ needs.start.outputs.dry_run }}
+          MAS_VERSION: ${{ needs.start.outputs.mas_version }}
+        run: |
+          DRY_RUN=${DRY_RUN} \
+            script/brew_tap_update ${MAS_VERSION}
+
+      - name: 🚀 Upload Bottles
+        env:
+          GH_TOKEN: ${{ github.token }}
+          DRY_RUN: ${{ needs.start.outputs.dry_run }}
+          MAS_VERSION: ${{ needs.start.outputs.mas_version }}
+        if: ${{ needs.start.outputs.dry_run == 'false' }}
+        run: |
+          gh release upload ${MAS_VERSION} \
+            .build/bottles/mas-*.bottle.tar.gz
+
+  homebrew-core:
+    runs-on: macos-15
+    needs: [start, prepare-release, homebrew-tap]
+    env:
+      PRE_RELEASE: ${{ needs.start.outputs.pre_release }}
```

I assume this syntax will work.
```suggestion
    if: ${{ needs.start.outputs.pre_release }} == 'false'
```

---

I suspect it will.